src/ghdldrv/ghdlsynth.adb

```diff
@@ -128,6 +128,10 @@ package body Ghdlsynth is
       P ("    Neither synthesize assert nor PSL");
       P ("  --no-assert-cover");
       P ("    Cover PSL assertion activation");
+      P ("  --assert-assumes");
+      P ("    Treat all PSL assertions like PSL assumes");
+      P ("  --assume-asserts");
+      P ("    Treat all PSL assumes like PSL assertions");
```

Comment:
```suggestion
      P ("    Treat all PSL asserts like PSL assumes");
      P ("  --assume-asserts");
      P ("    Treat all PSL assumes like PSL asserts");
```
(otherwise it would have to be "assumptions" too)

Reply:
Changed as suggested.